admin/starter/tests/database/ExampleDatabaseTest.php

```diff
@@ -31,6 +31,7 @@ public function testSoftDeleteLeavesRow()
         $this->setPrivateProperty($model, 'useSoftDeletes', true);
         $this->setPrivateProperty($model, 'tempUseSoftDeletes', true);
 
+        /** @var stdClass $object */
```

Comment:
`stdClass` should be imported or use `\stdClass` instead.

Reply:
The current sample test files has no namespace.

It seems better to have a namespace and can be autoloaded.